lib/managers/outgoingCall.ts

```diff
@@ -345,6 +352,13 @@ const onEnded = (evt: SessionEndedEvent): void => {
     Plivo.log.debug(`Outgoing call ended - ${cs._currentSession.callUUID}`);
     Plivo.log.info('Outgoing call ended');
     cs._currentSession.onEnded(cs, evt);
+
+    //  logout if logged in by token and token get expired
+    if(cs.isAccessToken && cs.accessToken == null) {
+      cs.emit('onLogout', 'ACCESS_TOKEN_EXPIRED');
```

Comment:
We must not have "Access token expired" message until we get the response from the server.

Reply:
this msg was emitted on the below scenrios, 
> only token was expired but call currently going on
> and after the user hangup the call this condition was executed

so as server already emits token expired msg